packages/lib/src/core/core.ts

```diff
@@ -42,6 +42,7 @@ class Core {
                 .setupSession(this.options)
                 .then(sessionResponse => {
                     const amount = this.options.order ? this.options.order.remainingAmount : sessionResponse.amount;
+                    this.session.configuration = sessionResponse.configuration
```

Comment:
```suggestion
                    this.session.configuration = sessionResponse.configuration;
```

Reply:
Surprise that the linter didn't catch this